extension/assets/.devcontainer/axon-ivy/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Axon Ivy Dev Container",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "devcontainer",
+	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+	"forwardPorts": [8080],
+	"containerEnv": {
+		"ENGINE_PORT": "8080"
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/java:1": {
```

Comment:
With this feature enabled it looks like java 21 is also installed additionally to java 17 which comes with the base image!